modules/olmv1-creating-a-service-account.adoc

```diff
@@ -0,0 +1,138 @@
+// Module included in the following assemblies:
+//
+// * extensions/ce/managing-ce.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="olmv1-creating-a-service-account_{context}"]
+= Creating a service account to manage cluster extensions
+
+Unlike {olmv0-first}, {olmv1} does not have permissions to install, update, and manage cluster extensions. Cluster administrators must create a service account and assign the role-based access controls (RBAC) required to install, update, and manage cluster extensions.
+
+[IMPORTANT]
+====
+include::snippets/olmv1-known-issue-service-accounts.adoc[]
+====
+
+.Prerequisites
+
+* Access to an {product-title} cluster using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Create a service account, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: <extension>-installer
+  namespace: <namespace>
+----
++
+.Example `extension-service-account.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pipelines-installer
+  namespace: pipelines
+----
+====
+
+. Apply the service account by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f extension-service-account.yaml
+----
+. Create a cluster role and assign RBAC, similar to the following example:
++
+[WARNING]
+====
+The following cluster role does not follow the principle of least privilege. This cluster role is intended for testing purposes only. Do not use it on production clusters.
+====
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <extension>-installer
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+----
++
+.Example `pipelines-cluster-role.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pipelines-installer
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
+----
+====
+
+. Add the cluster role to the cluster by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f pipelines-role.yaml
+----
+
+. Bind the permissions granted by the cluster role to the service account by creating a cluster role binding, similar to the following example:
++
+[source,yaml]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: <extension>-installer-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: <extension>-installer-clusterrole
```

Comment:
@michaelryanpeter per https://github.com/openshift/openshift-docs/pull/82245/files#diff-e44e70381349dd00e73c99c547be95941927faeaf5958930c49b641c4db8c02dR65, it is `<extension>-installer`, not `<extension>-installer-clusterrole`, so better to make it consistent.

Reply:
I decided to update Line 65 to have a difference between the service account name and the cluster role name. 

Addressed here: https://github.com/openshift/openshift-docs/pull/82245/commits/80d0396c13a630304ad3ba5b87ed6a0e5222e42c#diff-e44e70381349dd00e73c99c547be95941927faeaf5958930c49b641c4db8c02dR65